gcp/modules/gke-cluster/main.tf

```diff
@@ -12,10 +12,13 @@ module "gke_cluster" {
 
   initial_node_count = 1
   node_type          = "n1-standard-2"
-  kubernetes_version = "1.9.7-gke.6"
+  kubernetes_version = "1.10.7-gke.2"
```

Comment:
Do we need 1.10 for stackdriver beta? Just that it's kind of unrelated otherwise and I would prefer to leave it out if it's not needed and perhaps do as separate PR.

Reply:
Yes, 1.10 (is required)[https://cloud.google.com/monitoring/kubernetes-engine/installing] for Stackdriver Beta.